msyncd/synchronizer.cpp

```diff
@@ -184,7 +186,7 @@ bool Synchronizer::initialize()
         else
         {
             QObject::connect(&iSyncOnChangeScheduler, SIGNAL(syncNow(QString)),
-                             this, SLOT(startSync(QString)),
+                             this, SLOT(startScheduledSync(QString)),
```

Comment:
I'm not certain that that's true.  I think it's worth having semantic separation in the API.

But for now, let's just go with this.  We can revisit later if we decide that there is indeed value in being able to handle sync differently based on the reason for the trigger.